support-frontend/assets/components/page/page.jsx

```diff
@@ -16,19 +16,26 @@ type PropTypes = {|
   header: Node,
   footer: Node,
   children: Node,
-  classModifiers: Array<?string>
```

Comment:
unrelated to your PR, but I'm wondering why we have an `Array<?string>` here. Does this mean `classModifiers` is mandatory but the array can contain undefined/null values?

Reply:
It's the type expected by the function that puts together the BEM classes